src/main.py

```diff
@@ -1,46 +1,43 @@
-from typing import Final, Set
+from typing import Final
 from pathlib import Path
 
+import click
 import os
 
 from src.bazelize_deps import bazelize_deps
-from src.package import PackageMetadata
-from src.get_package_version import get_package_version
+from src.read_input_file import read_input_file
 
 BAZEL_WORKSPACE_DIR: Final = os.environ["BUILD_WORKSPACE_DIRECTORY"]
```

Comment:
The env var is only set if you run main.py with `bazel run`, if a `bazel test` imports this file it will cause an import error.
How about something like
```suggestion
BAZEL_WORKSPACE_DIR: Final = os.environ.get("BUILD_WORKSPACE_DIRECTORY") or os.environ.get("TEST_UNDECLARED_OUTPUTS_DIR") or os.environ.get("TEST_TMPDIR")
```
(the second fallback is because TEST_UNDECLARED_OUTPUTS_DIR is not guaranteed to be set, see https://bazel.build/reference/test-encyclopedia#initial-conditions)

I guess it would be best to move this into a function and call it to set the constant. It might also make sense to not make this a constant at all but to take it as CLI param instead, and just using the BUILD_WORKSPACE_DIRECTORY / TEST_UNDECLARED_OUTPUTS_DIR / TEST_TMPDIR values as default.